src/_includes/footer.njk

```diff
@@ -1,18 +1,38 @@
-<footer class="pb-3 w-100" id="footer">
+<footer class="footer w-100" id="footer">
     <div class="footer-top">
         <div class="container">
             <div class="row">
                 <div class="col-md">
-                    <h2 class="font-bold mb-1 text-[18px]">Links</h2>
+                    <h2 class="font-bold mb-1 text-[16px]">Links</h2>
                     <a href="/accessibility" class="block mt-1 hover:text-slate-200">Accessibility statement</a>
                     <a href="/site-map" class="block mt-1 hover:text-slate-200">Site map</a>
                 </div>
                 <div class="col-md">
-                    <h2 class="mb-1 mt-4 text-white font-bold text-[18px] mt-md-0">Contact Us</h2>
+                    <h2 class="mb-1 mt-4 text-white font-bold text-[16px] mt-md-0">Get in touch</h2>
                     <p>
                         <strong>Email:</strong>
-                        <a href="mailto:i-dot-ai-enquiries@cabinetoffice.gov.uk" class="inline-block hover:text-slate-200">i-dot-ai-enquiries@cabinetoffice.gov.uk</a>
+                        <a href="mailto:i-dot-ai-enquiries@cabinetoffice.gov.uk" class="inline-block ml-1 hover:text-slate-200">i-dot-ai-enquiries@cabinetoffice.gov.uk</a>
                     </p>
+                    <div class="footer__social-media-container">
+                        <a class="footer__social-media-link hover:text-slate-200" href="https://uk.linkedin.com/company/i-dot-ai">
+                            <svg width="24" height="25" viewBox="0 0 24 25" fill="none" aria-hidden="true" focusable="false">
+                                <path d="M19.7708 20.9663H16.3344V15.2573C16.3344 13.8959 16.3083 12.1439 14.5436 12.1439C12.7519 12.1439 12.4782 13.6252 12.4782 15.1568V20.9663H9.04177V9.22632H12.3429V10.8266H12.3873C12.8486 9.90394 13.9702 8.93005 15.6459 8.93005C19.1278 8.93005 19.7718 11.3596 19.7718 14.5222L19.7708 20.9663ZM5.16051 7.61991C4.05434 7.61991 3.16573 6.67063 3.16573 5.50299C3.16573 4.33637 4.05531 3.38811 5.16051 3.38811C6.26281 3.38811 7.15625 4.33637 7.15625 5.50299C7.15625 6.67063 6.26184 7.61991 5.16051 7.61991ZM6.88358 20.9663H3.43744V9.22632H6.88358V20.9663ZM21.49 0H1.71243C0.765809 0 0 0.793463 0 1.77248V22.831C0 23.8111 0.765809 24.6035 1.71243 24.6035H21.4871C22.4328 24.6035 23.2063 23.8111 23.2063 22.831V1.77248C23.2063 0.793463 22.4328 0 21.4871 0H21.49Z" fill="currentColor"/>
+                            </svg>
+                            <span class="visually-hidden">Linked In</span>
```

Comment:
```suggestion
                            <span class="visually-hidden">LinkedIn</span>
```

Reply:
Because it's visually hidden, I think the version with a space is likely to read better for screen readers.